components/common/BraveCoreThemeProvider.tsx

```diff
@@ -23,13 +23,28 @@ export default class BraveCoreThemeProvider extends React.Component<Props, State
     if (props.initialThemeType) {
       this.state = themeTypeToState(props.initialThemeType)
     }
-    chrome.braveTheme.onBraveThemeTypeChanged.addListener(this.setThemeState)
+    // Ensure we have access to braveTheme before updating.
+    // Otherwise this would break Storybook.
+    if (chrome.braveTheme) {
+      chrome.braveTheme.onBraveThemeTypeChanged.addListener(this.setThemeState)
+    }
   }
 
   setThemeState = (themeType: chrome.braveTheme.ThemeType) => {
     this.setState(themeTypeToState(themeType))
   }
 
+  componentDidUpdate (prevProps: Props) {
+    // Update theme based on React prop changes.
+    // This only runs on storybook and is needed
+    // since it has no access to chrome.* APIs
+    if ((chrome as any).braveTheme === undefined) {
```

Comment:
needed to add `chrome as any` here and in other places because TS doesn't recognize it, so compiler was saying this would always evaluate to false. Hopefully we can remove once we figure out a good way to have TS definitions (which I'm working in parallel).

Reply:
Aren't the definitions loaded from `tsconfig-storybook.json` in particular global.d.ts and chrom.d.ts?